From 2cd23ffb960641ebeff3c3071e0658d54c8d1a45 Mon Sep 17 00:00:00 2001
Message-Id: <2cd23ffb960641ebeff3c3071e0658d54c8d1a45.1522075821.git.jeremy@jcline.org>
From: Jeremy Cline <jeremy@jcline.org>
Date: Mon, 26 Mar 2018 10:47:22 -0400
Subject: [PATCH] Unpin the test requirements and use unittest2 for Python 3

unittest2 now supports Python 3 and the old unittest2py3k package is
abandoned: https://pypi.org/project/unittest2/.

Signed-off-by: Jeremy Cline <jeremy@jcline.org>
---
 social_core/tests/requirements-python2.txt | 4 ++--
 social_core/tests/requirements-python3.txt | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/social_core/tests/requirements-python2.txt b/social_core/tests/requirements-python2.txt
index 5ff9aa8..be3d83f 100644
--- a/social_core/tests/requirements-python2.txt
+++ b/social_core/tests/requirements-python2.txt
@@ -1,3 +1,3 @@
 -r requirements-base.txt
-unittest2==0.5.1
-mock==1.0.1
+unittest2
+mock
diff --git a/social_core/tests/requirements-python3.txt b/social_core/tests/requirements-python3.txt
index b41e8a2..ba79e41 100644
--- a/social_core/tests/requirements-python3.txt
+++ b/social_core/tests/requirements-python3.txt
@@ -1,2 +1,2 @@
 -r requirements-base.txt
-unittest2py3k==0.5.1
+unittest2
-- 
2.17.0.rc1

